util/test/programs.go

```diff
@@ -0,0 +1,67 @@
+package test
+
+// BoxApprovalProgram is a TEAL program which allows for testing box functionality
+const BoxApprovalProgram string = `#pragma version 8
```

Comment:
nit: this is a good candidate for `go embed` https://pkg.go.dev/embed

Reply:
resolving in favor of #1227